docs/tutorials/get-user-by-email.md

```diff
@@ -0,0 +1,73 @@
+# Tutorial: Get a user by email
+
+In this tutorial, you will learn how to get a user by their email address from the To-Do service.
```

Comment:
```suggestion
In this tutorial, you will learn how to get a user from the To-Do service by their email address.
```

Reply:
Included in my newer commmit: "Revised based on feedback from rbwatson"